### PR TITLE
Ticket/13841

### PR DESCRIPTION
**Pull request by Alf007**
Event added for the function move_topics() in
includes/functions_admin.php, so that you can insert any extension's table(s)
to be updated when topics are moved between forums

PHPBB3-13841

### CODE REVIEW

File: phpBB/includes/functions_admin.php [hunk 1]
```diff
@@ -534,6 +534,19 @@ function move_topics($topic_ids, $forum_id, $auto_sync = true)
...
+
+	/**
+	 * Perform additional actions before topics move
+	 *
+	 * @event core.move_topics_before_query
+	 * @var	array	table_ary	Array of tables from which forum_id will be updated for all rows that hold the moved topics
+	 * @since 3.1.5-RC1
+	 */
+	$vars = array(
+			'table_ary',
```
Comment (nickvergessen) [inline phpBB/includes/functions_admin.php]: same here.
Please add all method parameters and other relevant variables to the event.

**Comment by nickvergessen** [comment]
Contains commits from https://github.com/phpbb/phpbb/pull/3610 and therefor can not be merged before

**Comment by Alf007** [comment]
I don't know how the commit from #3610 get there ?! It should not as these are independant changes ?! ^^

**Comment by nickvergessen** [comment]
Merged via https://github.com/phpbb/phpbb/commit/7a55bcc0f3fc5a703b686affec413e891f558ff9

**Pull request closed without merging**